app/tests/test_browse_consignment.py

```diff
@@ -99,7 +99,7 @@ def test_browse_consignment_breadcrumb(
         "query_params, expected_results",
         [
             (
-                "date_filter_field=date_last_modified&date_from_day=01&date_from_month=05&date_from_year=2024",
+                "date_filter_field=date_last_modified&date_from_day=01&date_from_month=03&date_from_year=2024",
```

Comment:
This made me think....

do we have any tests for date in the future which raises error for being future? 

Either way for that type of test we should mock datetime so we have a fixed test where it wont change with the date it was run.

Reply:
added logic to mock date.today in test_date_validator.py for all test cases